### ISSUE TITLE
History command is empty when run again.

### ISSUE THREAD

**Issue by fscarmen** [kingToolbox/WindTerm#573]
Is there a history command ?

**Comment by haohaoget** [comment]
同，希望能增加一个显示输入过的历史命令的窗口

**Comment by joshatt** [comment]
Yeah. Hope it's like FinalShell, which automatically prevents identical items, and allows user to delete any item.

**Comment by fscarmen** [comment]
Although WindTerm has some advantages, I have given up using it for the second time because it has not been updated for a long time and the user experience is poor.
